src/python/base/utils.py

```diff
@@ -626,15 +626,21 @@ def restart_machine():
     os.system('sudo shutdown -r now')
 
 
-def search_string_in_file(search_string, file_handle):
-  """Helper to search for a string in a large binary file without memory
+def search_bytes_in_file(search_bytes, file_handle):
+  """Helper to search for bytes in a large binary file without memory
   issues.
   """
-  # TODO(aarya): This is too brittle and will fail if we have a very large line.
-  search_string = search_string.encode('utf-8')
-  for line in file_handle:
-    if search_string in line:
-      return True
+  old_pos = file_handle.tell()
```

Comment:
Do we have usecase where file_handle position is not zero. anyway good to do this.

Reply:
Not that I know of. I'll leave this here just in case.